test/verify/check-testlib

```diff
@@ -213,8 +213,8 @@ class TestSystemd(unittest.TestCase):
 
 
 @skipDistroPackage()
+@nondestructive
```

Comment:
Context: I originally started to write a new `TestTestlib.testOopsDetection`, but then discarded that -- it would do almost the same as `TestMenu.testBasic`, so I refined that instead. But this little detail is still legit IMHO, but doesn't warrant its own PR.